middle_end/flambda2/to_cmm/to_cmm_set_of_closures.ml

```diff
@@ -203,18 +203,24 @@ end = struct
         res,
         eff,
         updates )
-    | Function_slot { size; function_slot; last_function_slot } -> (
-      let code_id = Function_slot.Map.find function_slot decls in
-      let code_symbol =
-        R.symbol_of_code_id res code_id ~currently_in_inlined_body:false
-      in
-      let (kind, params_ty, result_ty), closure_code_pointers, dbg =
-        get_func_decl_params_arity env code_id
-      in
+    | Dummy_function_slot { last_function_slot } ->
```

Comment:
We're getting at the point in code size (no pun intended) where some of the cases of the matches below could be split into their own function.